support-frontend/assets/pages/contributions-landing/components/StripeCardForm/StripeCardForm.jsx

```diff
@@ -184,9 +184,22 @@ const CardForm = (props: PropTypes) => {
     }
   };
 
+  const recaptchaElementNotEmpty = (): boolean => {
+    const el = document.getElementById('robot_checkbox');
+    if (el) {
+      return el.children.length > 0;
+    }
+    return true;
+  };
+
   // Creates a new setupIntent upon recaptcha verification
   const setupRecurringRecaptchaCallback = () => {
     setCalledRecaptchaRender(true);
+    // Fix for safari, where the calledRecaptchaRender state handling does not work. TODO - find a better solution
```

Comment:
In the old days we'd inspect the element's offsetHeight to know if it was visible or not...